### PR TITLE
Beam 14082 update payground for mobile

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14082]
- Instead of Playground Iframe use image 
- When user clicks on the image opens Playground page (https://play.beam.apache.org/)

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: website/www/site/assets/scss/_playground.sass [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  .playground__mobile
+    display: flex
+
+@media (min-width: $mobile)
+  .playground__mobile
+    display: none
+
+.center
+  display: flex
+  justify-content: center
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/assets/scss/_playground.sass]: done

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

**Comment by dariabezkorovaina** [comment]
@pcoet please take a look:)

File: website/www/site/assets/scss/_playground.sass [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@import "media"
+
+.playground-section
+  padding: $pad-l $pad;
```
Comment (pcoet) [inline website/www/site/assets/scss/_playground.sass:21]: SASS doesn't require semicolons, right?

**Review by pcoet — APPROVED** [review]
One nit, but otherwise LGTM.

File: website/www/site/assets/scss/_playground.sass (hunk 2 above)
Comment (AydarZaynutdinov) [reply website/www/site/assets/scss/_playground.sass]: Changed.

**Review by pcoet — APPROVED** [review]

**Comment by pabloem** [comment]
LGTM thanks all!

**Pull request merged**